app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,24 @@
+class NotesController < ApplicationController
+
+def index
+  @notes = Note.all
+  render json: @notes
+end
+
+
+def create
+  @note = Note.new(note_params)
+    if @note.save
+      render json: @note
+    else
+      render json: @note.errors
+    end
+end
+
+
+def note_params
+  params.permit(:title, :body, :tags)
```

Comment:
You actually don't want `:tags` here - because you don't want a comma-separated string passed into the Note.new (hence, the error you were seeing.

Instead, after it's saved, you can take `params[:tags]`, split it on the commas (so it becomes an array) and then tag @note with each one of those tags (by find_or_initializing a tag by the name and pushing it into @note.tags).